.travis.yml

```diff
@@ -7,30 +7,24 @@ dist: trusty
 
 env:
   matrix:
-    - xGSL=0 xMPI=0 xPYTHON=0
-    - xGSL=0 xMPI=0 xPYTHON=1
-    - xGSL=0 xMPI=1 xPYTHON=0
-    - xGSL=0 xMPI=1 xPYTHON=1
-    - xGSL=1 xMPI=0 xPYTHON=0
-    - xGSL=1 xMPI=0 xPYTHON=1
-    - xGSL=1 xMPI=1 xPYTHON=0
-    - xGSL=1 xMPI=1 xPYTHON=1
+    - xGSL=0 xMPI=0 xPYTHON=1 CACHE_NAME=JOB
```

Comment:
What does the CACHE_NAME mean here? I cannot see it used anywhere in the script. And could there be a problem with using the same cache name on multiple processes running simultaneously?